spec/steps/execute.fmf

```diff
@@ -133,7 +133,7 @@ description: |
       - /tmt/steps/execute/internal.py
       - /tmt/steps/execute/detach.py
 
-    /simple:
+    /Simplest usage:
         summary: Simple use case should be super simple to write
```

Comment:
I'm not sure about using spaces and capital chars as part of the fmf name:

```
> tmt story ls spec/steps/execute
/spec/steps/execute/detach
/spec/steps/execute/isolate
/spec/steps/execute/shell/Multi-line script
/spec/steps/execute/shell/Multiple commands
/spec/steps/execute/shell/Simplest usage
/spec/steps/execute/tmt
```

This would make the naming inconsistent.

Reply:
Well, I am trying to have nice headers in e.g. https://tmt.readthedocs.io/en/latest/spec/steps.html#multi
Might be just me, but single name and lower case is for me hard to distinguish from options (shell vs multi) and these paragraphs explain possibilities of an option, not the option itself.

Reply:
@psss: How about using `fmf name` only for option names and executor names? So examples how to use script will be part of 'script' documentation (so no 'shell' there anymore)

Reply:
I see. Understand the use case. When exporting to online docs sometimes one would like to choose a better title for the section. Hm, what about a new optional `title` attribute for user stories? If defined it would be used during the `rst` export.

Reply:
I liked the `title` idea. Added and renamed short names. Now it is:
```
> tmt story ls spec/steps/execute
/spec/steps/execute/detach
/spec/steps/execute/isolate
/spec/steps/execute/script/multi
/spec/steps/execute/script/several
/spec/steps/execute/script/simple
/spec/steps/execute/tmt
```

Reply:
The other commit in this branch is an attempt to implement 'title'

Reply:
Please check, @psss . Locally it works :)

Reply:
Nice, thanks!